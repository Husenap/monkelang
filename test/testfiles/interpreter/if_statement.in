monke has number bananas is 10

monke think bananas is greater than 5 monke do
  monke say "too many bananas"
monke change mind
  monke say "nice bananas is " + bananas
monke thinked

bananas is 4

monke think bananas is greater than 5 monke do
  monke say "too many bananas"
monke change mind
  monke say "nice bananas is " + bananas
monke thinked

bananas is 5

monke think bananas is greater than 5 monke do
  monke say "too many bananas"
monke think again bananas is equal to 5 monke do
  monke say "perfect amount of bananas"
monke change mind
  monke say "need more bananas"
monke thinked

