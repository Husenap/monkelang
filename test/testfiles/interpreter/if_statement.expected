too many bananas
nice bananas is 4
perfect amount of bananas
